app/router/session.py

```diff
@@ -1,173 +1,46 @@
-            current_date,
-            current_timestamp,
-            repeated_session_start_datetime,
-        ) = build_datetime_and_timestamp(start_time)
-        if session_start_date <= current_date:
-            if repeat_schedule is None:
-                return (
-                    session_start_timestamp
-                    <= current_timestamp + session_start_buffer_time
-                )
-            else:
-                return (
-                    repeated_session_start_datetime
-                    <= current_timestamp + session_start_buffer_time
-                )
-        return False
-    return True
-
-
-def has_session_ended(end_time: str, repeat_schedule: str):
-    """
-    Checks if session has ended
-    - If end time is given:
-        - If session end date is greater than or equal to current date:
-            - If session is not repeating, checks if the current timestamp is less than or equal to session end timestamp
-            - If session is repeating, builds a timestamp using current date and session end time and checks that with current timestamp
-    - Else, always returns True
-    """
-    if end_time is not None:
-        (
-            session_end_date,
-            session_end_timestamp,
-            current_date,
-            current_timestamp,
-            repeated_session_end_datetime,
-        ) = build_datetime_and_timestamp(end_time)
-        if session_end_date >= current_date:
-            if repeat_schedule is not None:
-                return current_timestamp <= repeated_session_end_datetime
-            return current_timestamp <= session_end_timestamp
-        return False
-    return True
-
-
-def is_session_repeating(repeat_schedule: str):
-    """
-    Checks if the repeating schedule matches to the current day.
-    """
-    if repeat_schedule is not None:
-        if repeat_schedule["type"] == "weekly":
-            return datetime.today().weekday() in repeat_schedule["params"]
-    return True
-
-
-@router.get("/{session_id}", response_model=SessionResponse)
-async def get_session_data(session_id: str):
-    """
-    This API returns session details corresponding to the provided session ID, if the ID exists in the database and if the session is open
+    This API returns session details corresponding to the provided session ID, if the ID exists in the database
 
     Parameters:
     session_id (str): The ID against which details need to be retrieved
 
     Returns:
-    list: Returns session details if the session ID exists in the database and the session is open, false if the session is closed. If the ID does not exist, 404 is returned
+    list: Returns session details if the session ID exists in the database. If the ID does not exist, 404 is returned
 
     Example:
     > $BASE_URL/session/1234
     returns [{session_data}]
 
-    > $BASE_URL/session/{closed_session_id}
-    returns false
-
     > $BASE_URL/session/{invalid_id}
     returns {
```

Comment:
Is this being returned now or `raise`d?

Reply:
Its being raised. Will make the change. The frontend handles this exception.